src/components/ProductCard/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { StarRate, Wishlist, Basket } from "@shared";
+import { Link } from "react-router-dom";
+
+interface ProductProp {
+  img: string;
+  name: string;
+  type: string;
+  cost: number;
+  rate: number;
+}
+
+const ProductCardExample: ProductProp = {
+  img: "https://www.collinsdictionary.com/images/full/paper_111691001.jpg",
+  name: "Paper name",
+  type: "Paper type",
+  cost: 1500,
+  rate: 1,
+};
+
+export default function ProductCard ({ img, name, type, cost, rate }: ProductProp & typeof ProductCardExample): JSX.Element {
+  return (
+    <div className="m-1 h-[438px] w-80 rounded-lg border-2 border-black">
+      <div className="relative">
+        <Link to={"/catalog"}>
```

Comment:
Let's use <picture> for our images.
Read more: https://www.w3schools.com/tags/tag_picture.asp

Reply:
Can not see lazy loading for images. Can u record how it works in your case?

Reply:
Okey, approve this but in the future I think it will be better to obtain from backend 2 images - one extremely small and one with normal size. And first we will show small version of image and after we load our original image - show it for user. Example u can see here: https://www.youtube.com/watch?v=hJ7Rg1821Q0
Maybe we can add this like task into Jira for both back- and frontend.